examples/getCiphers/getCiphers.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/bmc-toolbox/bmclib/v2/providers/ipmitool"
+	"github.com/bombsimon/logrusr/v2"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Minute)
+	defer cancel()
+
+	// set BMC parameters here
```

Comment:
I suggest using the same format as the other [examples]( https://github.com/bmc-toolbox/bmclib/tree/main/examples/inventory)

- Add a doc.go with some notes about the example.
- Add an output.txt or such with the output returned by this example.
- Accept flags instead for parameters.

Reply:
I modeled mine after reset_bmc.go. I'll rework it to model after inventory instead, thanks!